### DISCUSSION TITLE
adding role in ticket support with /add

### DISCUSSION THREAD

**Discussion by sadoj-kuro** [discord-tickets/bot#355] [General]
currently we can add a member with /add but what would be good is to also be able to add a group of people who have a role

**Comment by eartharoid** [comment]
Could you give an example of when this might be useful?

**Reply by sadoj-kuro** [reply]
I personally use the bot for uses on GTA RP. tickets are used to connect different groups together for transactions. Being able to invite a role rather than members one by one saves us time. To be able to invite all the members of a "support" role rather than one by one or even to be able to invite all the people of the same role who could be concerned.

sorry my english is bad im french ^^

**Reply by eartharoid** [reply]
>  To be able to invite all the members of a "support" role rather than one by one

This is what each ticket category's `roles` (`staffRoles`) property is for - so staff are added automatically when the ticket is created.
